docs/general-information/project-template.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: default
+title: Project Template
+parent: General Information
+---
+
+# Project Template
+
+## Overview
+
+The project template is a base project with all the basic functionality built in for developing
+Python or R projects. This helps save time and effort at the beginning of a project and keeps
+project development workflows and standards consistent across the lab. The base project uses
+[Docker](https://ndclab.github.io/wiki/docs/technical-docs/docker-usage.html) as a development
+platform and as a deployment platform.
+
+By default the project contains a project directory, markdown templates for issues and pull
+requests, shell scripts to automate some basic actions, and yaml configurations for [GitHub
+Actions].
+
+## Usage
+
+To use the template first clone the project template
+[repository](https://github.com/NDCLab/project-template/tree/main).
+
+Once the repository has been cloned, open a terminal and `cd` into the directory.
+
+Run the `init.sh` [Bash](https://en.wikipedia.org/wiki/Bash_%28Unix_shell%29) script:
+
+```bash
+# "name" is the name of the project
+# "name" should be all lower case, and contain no spaces
+./init.sh name
+```
+
+To create the [Docker container](https://www.docker.com/resources/what-container), run
+`docker-compose up -d --build`. This will run the [Docker Compose](https://docs.docker.com/compose/)
+configuration file and automatically build the Docker container and run it.
```

Comment:
After this part, say "See [docker doc] for a description of docker usage"

Again, the idea is not to refer to other docs every time you mention something covered in another doc, just the first time you refer to something covered in another doc. So, the prior comment tells people about the main github doc, here is the first time you talk about docker in this doc, so you refer to the docker doc here. Later on in this doc, as you continue to refer to things covered in either the github-main doc or the docker doc, you should NOT continue to refer to those docs, as you have already made one reference to each of them.